envoy/admin/v2/admin.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package envoy.admin;
+
+import "google/protobuf/any.proto";
+import "envoy/api/v2/rds.proto";
+
+import "gogoproto/gogo.proto";
+
+// [#protodoc-title: ConfigDump]
+
+// The /config_dump admin endpoint uses this wrapper message to maintain and serve arbitrary
+// configuration information from any component in Envoy.
+message ConfigDump {
```

Comment:
I don't think we need to do it right now, but in the future I think we want to make each endpoint a full RPC with a request and response message. This will allow us to serve both binary gRPC/REST and take parameters, etc. Can you potentially add a TODO here to that effect?

Reply:
👍 Yup